Khan/wonder-blocks
# WB-1119: Report code coverage to codecov and chromatic changes on push to master

## Summary:
- Renames `node-ci.yml` to `node-ci-pr.yml` to separate the event types for the workflows.
- Adds `node-ci-push.yml` to perform the following jobs after landing into `master`:
    - Report code coverage to codecov.
    - Report and auto accept visual tests on Chromatic (hopefully this fixes an issue that presented some inconsistent changes when creating new builds).

Issue: WB-1119

## Test plan:

- Merge this PR.
- Verify that we start reporting to Codecov.
- Verify that Chromatic uses this build as the new baseline.
